### PR TITLE
Drop vendoring

### PR DESCRIPTION
**Pull request by MartinWeindel**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement

**What this PR does / why we need it**:
Remove vendoring from project

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other developer
Remove vendoring from project
```

### CODE REVIEW

**Review by ScheererJ — APPROVED** [review]
Thanks a lot for the change.

/lgtm

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -18,44 +18,43 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# We need to explicitly pass GO111MODULE=off to k8s.io/code-generator as it is significantly slower otherwise,
-# see https://github.com/kubernetes/code-generator/issues/100.
-export GO111MODULE=off
```
Comment (timuthy) [inline hack/update-codegen.sh:23]: Why is this removed?

**Review by timuthy — COMMENTED** [review]
Just one question.

File: hack/update-codegen.sh (hunk 1 above)
Comment (MartinWeindel) [reply hack/update-codegen.sh]: It didn't feel so slow without, but after measuring it is 44s vs 21s on my machine.
Added it again. Thanks for the hint.
